### PR TITLE
[COMP] Kettering

### PR DESCRIPTION
**Pull request by MattD8957**
End of COMP-Kettering

### CODE REVIEW

File: src/main/java/frc/robot/command/Collect.java [hunk 1]
```diff
@@ -4,21 +4,25 @@
...
 	 * @param collector subsystem
+	 * @param indexer subsystem
 	 */
-	public Collect(DoubleSupplier powerSupplier, Collector collector) {
+	public Collect(DoubleSupplier powerSupplier, Collector collector, Indexer indexer) {
 		this.collector = collector;
+		this.indexer = indexer;
 		this.powerSupplier = powerSupplier;
 
 		addRequirements(collector);
```
Comment (WindowsVistaisCool) [inline src/main/java/frc/robot/command/Collect.java:28]: does indexer need requirements

**Review by WindowsVistaisCool — APPROVED** [review]
something we need to think abt for the future is a variable way to set smartcollect power. otherwise lgtm if you want to merge or not

**Comment by WindowsVistaisCool** [comment]
btw - most week 0 issues have been moved to ann arbor milestone

**Comment by MattD8957** [comment]
@Jeffthedoor Tag or release please

**Comment by WindowsVistaisCool** [comment]
release created you can edit it if needed

**Pull request merged**